js/components/controlBar.js

```diff
@@ -245,9 +245,11 @@ var ControlBar = React.createClass({
         </span>
       </div>,
 
-      "live": <div className="live" style={InlineStyle.controlBarStyle.liveItemStyle}>
-        <div style={InlineStyle.controlBarStyle.liveCircleStyle}></div>
-        <div style={InlineStyle.controlBarStyle.liveTextStyle}>{liveText}</div>
+      "live": <div className="live" style={InlineStyle.controlBarStyle.controlBarItemSetting}>
+        <div style={InlineStyle.controlBarStyle.liveItemStyle}>
+          <div style={InlineStyle.controlBarStyle.liveCircleStyle}></div>
+          <span style={InlineStyle.controlBarStyle.liveTextStyle}> {liveText}</span>
```

Comment:
liveText should be using localization text file instead of hard-coded in line 239